src/System.Security.Permissions/src/System/Security/CodeAccessPermission.cs

```diff
@@ -11,14 +11,14 @@ public abstract partial class CodeAccessPermission : IPermission, ISecurityEncod
         public void Assert() { }
         public abstract IPermission Copy();
         public void Demand() { }
-        [Obsolete("Deny is obsolete and will be removed in a future release of the .NET Framework. See http://go.microsoft.com/fwlink/?LinkID=155570 for more information.")]
+        [Obsolete]
```

Comment:
I figured if you're using this API it's already not working, obsolescence is a lesser concern. How are obsolete attributes relevant in implementation assemblies?